### PR TITLE
Update japanese translation.

### PR DESCRIPTION
**Pull request by niyari**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Internationalization and localization
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #327

## What is the new behavior?

- Update japanese translation.

## Other information
TODO : Eliminate shaky notation. Concise description.

## Quality check

Before creating this PR, have you:

- [x] Followed the code style guideline as described in [CONTRIBUTING.md](https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md)
- [x] Verified that the change work in Release build configuration
- [x] Checked all unit tests pass

### CODE REVIEW

**Comment by veler** [comment]
Hello,
Thanks for updating the translation! @inductor, would you like to review this one?

File: src/dev/impl/DevToys/Strings/ja-JP/JsonFormatter.resw [hunk 1]
```diff
@@ -136,7 +136,7 @@
     <value>入力</value>
   </data>
   <data name="Minified" xml:space="preserve">
-    <value>軽量化</value>
+    <value>ミニファイ</value>
```
Comment (inductor) [inline src/dev/impl/DevToys/Strings/ja-JP/JsonFormatter.resw:139]: Do you think minify is a common term? Personally I understand but I am not sure 🤔

Comment (niyari) [reply src/dev/impl/DevToys/Strings/ja-JP/JsonFormatter.resw:139]: Change to `縮小`. (to conform to the notation used in MSDN)

**Review by inductor — APPROVED** [review]

**Comment by btiteux** [comment]
Thank you for your contribution.

**Pull request merged**